esphome/components/image/__init__.py

```diff
@@ -259,6 +261,18 @@ def load_svg_image(file: bytes, resize: tuple[int, int]):
     return Image.open(io.BytesIO(svg_image))
 
 
+def invert_image_colors(image):
+    from PIL import Image
+
+    r, g, b, a = image.split()
+
+    def invert(image):
+        return image.point(lambda p: 255 - p)
+
+    r, g, b = map(invert, (r, g, b))
+    return Image.merge(image.mode, (r, g, b, a))
```

Comment:
Why not use pillow to do the inversion, instead of reinventing the wheel?
```suggestion
    import PIL.ImageOps
    
    return PIL.ImageOps.invert(image)
```

Reply:
This needs some testing though; it might fail for RGBA images